.babelrc

```diff
@@ -1,3 +1,14 @@
 {
-  "presets": ["@babel/preset-env"]
+  "presets": [
+    [
+      "@babel/preset-env",
```

Comment:
This is default babel configuration for dev and production for targeting "modern" browsers (versions that support ES Modules )
https://babeljs.io/docs/en/babel-preset-env#targetsesmodules